hopsworks-common/src/main/java/io/hops/hopsworks/common/featurestore/online/OnlineFeaturestoreFacade.java

```diff
@@ -35,6 +37,7 @@
  * Online-Featurestore has its own JDBC Connection Pool defined in persistence unit `featurestorePU`
  */
 @Stateless
+@TransactionAttribute(TransactionAttributeType.REQUIRES_NEW)
```

Comment:
@SirOibaf is this because we are using a native query?

Reply:
Opsie dopsie. I wanted to annotate the class with the default `TransactionAttribute` but I ended up picking the wrong one.

Reply:
Fixed

Reply:
But since `TransactionAttributeType.REQUIRED` is the default, then why annotate the class with it?

Reply:
Because I like things to be explicit.